workflow-templates/im-test-k6-manual-advanced.yml

```diff
@@ -0,0 +1,66 @@
+name: Manual trigger K6 test - Advanced
+
+on: 
+  workflow_dispatch:
+    inputs:
+      envToTest:
+        description: 'Environment to run test against [[1] dev, [2] qa, [3] stage]'   
+        required: true
+        default: '1'
+      testFolderRoot:
+        description: 'Test folder root directory from root of repo'
+        required: true
+        default: "tests/k6"
+      testFile:
+        description: 'K6 test file to run'     
+        required: true
+      k6_extra_params:
+        description: 'K6 command extra paramaters'
+        required: false
+        default: '' # You can set default paramaters to run on every K6 test.
+
+jobs:
+  k6_test:
+    runs-on: [self-hosted, ubuntu-20.04]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Setup K6
+      uses: im-open/setup-k6-perf-testing@v1.0.0
+      with:
+        version: 0.33.0
```

Comment:
This should have a todo or an ENV at the top asking the user to verify their version.

Reply:
Added todo.